docs/ref/contrib/staticfiles.txt

```diff
@@ -323,6 +323,12 @@ For example, the ``'css/styles.css'`` file with this content:
 
     @import url("../admin/css/base.27e20196a850.css");
 
+.. admonition:: Calculation of integrity
+
+    Keep these changes in static files in mind when using the ``integrity``
+    attribute for these paths: The checksum must be calculated after the
+    changes are made, e.g. after they have been collected by ``collectstatic``.
```

Comment:
I would re-work this paragraph to make it more explicit about the intent of the admonition, specifically I think the message has to be explicit about how the integrity usage is optional and is separated from the hashes the storage calculates:

```
    If the optional ``integrity`` attribute of the ``<script>`` tag is used,
    the value of its checksum  should be calculated after the resources have
    been collected by ``collectstatic``.
```